docs/application/dotnet/guides/multimedia/webrtc.md

```diff
@@ -0,0 +1,506 @@
+                break;
+        }
+    }
+    ...
+    await webRtc.StartAsync();
+    // After finishing negotiation, TrackAdded event could be invoked if receiving audio/video data from the remote peer exists
+    ```
+    > [!IMPORTANT]
+    > `ApplyAudioStreamPolicy()` or `Display` must be called inside of the callback set by the `TrackAdded` event handler, if you want to output the audio or video track from the remote peer to the local target device's audio device or video display.
+
+2. To get a media packet handle that packs the audio or video data from a remote peer, use the `AudioFrameEncoded` or `VideoFrameEncoded` event:
+
+    ```csharp
+    webRtc.AudioFrameEncoded += (s, e) =>
+    {
+        // Use media packet - copy it's data or pass it to another API.
+        // Unref MediaPacket after use.
+    }
+    webRtc.VideoFrameEncoded += (s, e) =>
+    {
+        // Use media packet - copy it's data or pass it to another API.
+        // Unref MediaPacket after use.
+    }
+    ...
+    await webRtc.StartAsync();
+    // After finishing negotiation, AudioFrameEncoded or VideoFrameEncoded event handler could be called if receiving audio/video data from the remote peer exists
+    ```
+
+3. To render sending audio/video data on the local target device, use `EnableAudioLoopback()` or `EnableVideoLoopback()`:
+
+    ```csharp
+    ...
+    var audioSource = new MediaMicrophoneSource();
+    var videoSource = new MediaCameraSource();
+
+    webRtc.AddSource(audioSource);
+    webRtc.AddSource(videoSource);
+
+    audioSource.EnableAudioLoopback(new AudioStreamPolicy(AudioStreamType.Media));
+    videoSource.EnableVideoLoopback(new Display(window));
+    ...
+    await webRtc.StartAsync();
+    ```
+
+4. To change display mode or display visibility, use `MediaStreamTrack.DisplayMode` or `MediaStreamTrack.DisplayVisible` property.
+
+    Three types of display modes exist, `WebRTCDisplayMode.LetterBox`, `WebRTCDisplayMode.OriginSize`, and `WebRTCDisplayMode.Full`.
+    These properties are also available for any `MediaStreamTrack` of video loopback.
+    In the following example code, it tries to change the mode to `WebRTCDisplayMode.OriginSize` after the display is set and change the visibility in an event function:
+
+    ```csharp
+    webRtc.TrackAdded += (s, e) =>
+    {
+        if (e.MediaStreamTrack.Type == MediaType.Video)
+        {
+            // You should create or get window instance to display video.
+            e.MediaStreamTrack.Display = new Display(window);
+            e.MediaStreamTrack.DisplayMode = WebRTCDisplayMode.OriginSize;
+            e.MediaStreamTrack.DisplayVisible = true;
+        }
+    }
+    ...
+    await webRtc.StartAsync();
+    ```
+
+<a name="statistics_info"></a>
+## Get statistics information
+
+You can get the statistics information of the WebRTC.
+
+1. To get statistics information, use the `WebRTC.GetStatistics()`. This method is possible in the `WebRTCState.Playing` state.:
```

Comment:
```suggestion
1. To get statistics information, use the `WebRTC.GetStatistics()`. This method is possible in the `WebRTCState.Playing` state:
```

Reply:
The change suggested in Line# 492 does not appear in the staging site. Please check again.